### ISSUE TITLE
Fetch immediately on src property assignment

### ISSUE THREAD

**Issue by josh** [github/include-fragment-element#4]
Content should be fetch immediately before the element is attached to any document. The replacement step can be deferred till the attach hook is fired.

Depends on #3 to hook into src changes.
Depends on #2 to have a promise to cache.

Prior art

``` js
img = document.createElement('img')
img.src = "preload-this.png"
// later...
document.body.appendChild(img)
```
